content/en/blog/_posts/2023-12-20-disabling-in-tree-cloud-provider-goes-beta.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: blog
+title: "Kubernetes 1.29: Cloud Provider Integrations Are Now Separate Components"
+date: 2023-12-20
+slug: cloud-provider-integration-changes
+---
+
+**Authors:** Michael McCune (Red Hat), Andrew Sy Kim (Google)
+
+For Kubernetes v1.29, you need to use additional components to integrate your
+Kubernetes cluster with a cloud infrastructure provider. By default, Kubernetes
+v1.29 does not run with the legacy, compiled-in cloud provider integrations
```

Comment:
I think we should reword this to:
```
By default, Kubernetes v1.29 disallows running the legacy compiled-in cloud provider integrations by default.
```

Reply:
i've updated the language a bit, but i did drop one of the "by default"s from that ;)